README.md

```diff
@@ -29,15 +29,63 @@ next steps. Note that you will need to know where your team stores their
 remote profile data in order to use
 [`aws-profile-sync`](https://github.com/cisagov/aws-profile-sync).
 
-To create the build user, follow these instructions:
+### Creating a Build User ###
```

Comment:
Just as a to-do we should switch the titles in the README to sentence case to follow our own [style guide](https://github.com/cisagov/development-guide/blob/develop/style-guide.md#headers-and-headings).